src/api/migraphx.py

```diff
@@ -170,6 +170,21 @@ def program(h):
              const=True)
 
 
+@auto_handle
+def onnx_options(h):
+    h.constructor('create')
+    h.method(
+        'add_parameter_shape',
```

Comment:
I think this should be named as `set_input_parameter_shape` or `set_parameter_shape` instead to make it clearer.

Reply:
changed the name